<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QMainWindow" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LZ77 Tool</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="userText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>49</y>
      <width>540</width>
      <height>191</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Type to compress or decompress.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compressButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Compress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decompressButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Decompress</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>258</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="decompressGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="decompressedLabel">
       <property name="text">
        <string>Decompressed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="decompressResult">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="processedTxtLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Processed Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="unprocessedTxtLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unprocessed Text</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>258</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="compressGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="compressedLabel">
       <property name="text">
        <string>Compressed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="compressResult">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
